### PR TITLE
Update menu icons to follow style guide

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
Use standard icons from style guide.

Comments
----------------------------------------
Let's fix this now before 5.7 is released.

### CODE REVIEW

File: CRM/Upgrade/Incremental/sql/5.7.alpha1.mysql.tpl [hunk 1]
```diff
@@ -6,7 +6,7 @@ UPDATE civicrm_navigation SET icon = "crm-i fa-address-book-o" WHERE name = "Con
 UPDATE civicrm_navigation SET icon = "crm-i fa-money" WHERE name = "Contributions" AND domain_id = {$domainID} AND icon IS NULL;
 UPDATE civicrm_navigation SET icon = "crm-i fa-calendar" WHERE name = "Events" AND domain_id = {$domainID} AND icon IS NULL;
 UPDATE civicrm_navigation SET icon = "crm-i fa-envelope-o" WHERE name = "Mailings" AND domain_id = {$domainID} AND icon IS NULL;
-UPDATE civicrm_navigation SET icon = "crm-i fa-users" WHERE name = "Memberships" AND domain_id = {$domainID} AND icon IS NULL;
+UPDATE civicrm_navigation SET icon = "crm-i fa-id-badge" WHERE name = "Memberships" AND domain_id = {$domainID} AND icon IS NULL;
```
Comment (seamuslee001) [inline CRM/Upgrade/Incremental/sql/5.7.alpha1.mysql.tpl]: @colemanw I think we will need an upgrade step in 5.7.beta1.mysql.tpl for anyone has already upgraded to the rc

**Comment by colemanw** [comment]
Thanks @seamuslee001 - updated.

FYI @kcristiano

**Comment by colemanw** [comment]
Let's hold off on merging this as I'll have a few more tweaks following up on https://github.com/civicrm/org.civicrm.styleguide/pull/22

**Comment by colemanw** [comment]
@seamuslee001 done. Now ready for review/merge.

**Comment by seamuslee001** [comment]
@colemanw this looks good, i'm tempted to say once we merge this the upgrade steps from the .beta1 should be applied in 5.8.alpha1 as well just as a fall back measure for anyone that is already on beta1 of 5.7 However this is good and I tested the upgrade and confirmed that the icons do change in the navigation as per the style guide adding merge on pass

**Pull request merged**